contrib/node/tests/python/pants_test/contrib/node/tasks/BUILD

```diff
@@ -1,6 +1,56 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+target(
```

Comment:
Use target tags, not aliases: can delete these three.

Instead, mark integration tests `tags={'integration'}` (see example further down in this file). Then you will be able to run tests with:
```
# Only integration
./pants --tag='integration' test contrib/node::
# Only unit
./pants --tag='-integration' test contrib/node::
# Everything
./pants test contrib/node::
```